push.sh

```diff
@@ -11,7 +11,11 @@ then
   echo
   echo "Pushing release commits and tag"
   echo "------------------------------------------------------------------------------"
-  git push && git push --tags
+  if ! (git push && git push --tags); then
```

Comment:
why previous code was not failing and this one will?
We have `set -e`

Reply:
No idea